src/binance-signer.ts

```diff
@@ -0,0 +1,35 @@
+import { bytesToHex } from './crypto-utils';
+import { IEthereumSigner } from './solidity-helpers'
+import ethutil from 'ethereumjs-util'
+const { crypto } = require('@binance-chain/javascript-sdk')
+/**
+ * Signs message using a Binance account.
+ * This signer should be used for interactive signing in the browser.
+ */
+export class BinanceSigner implements IEthereumSigner { 
```

Comment:
We should probably rename `IEthereumSigner` to `IMessageSigner`, but that can go in another PR.